common/src/main/java/com/wynntils/core/WynntilsMod.java

```diff
@@ -74,6 +74,10 @@ public static boolean postEvent(Event event) {
         }
     }
 
+    public static void postEventOnMainThread(Event event) {
```

Comment:
We could/should have replaced all other occurrences of this. Doesn't really matter though.

Reply:
Most places should *not* call this, I assume.